packages/context/src/notification/NotificationProvider.tsx

```diff
@@ -48,18 +51,14 @@ function NotificationProvider({ children }: { children: React.ReactNode }) {
                 const documentsConnectionInfo = await getConnectionInfo('documentChanges');
 
                 if (instancesConnectionInfo) {
-                    const options = {
-                        accessTokenFactory: async () => {
-                            if (instancesConnectionInfo.accessToken) {
-                                return instancesConnectionInfo.accessToken;
-                            } else {
-                                return getConnection();
-                            }
+                    const options: IHttpConnectionOptions = {
+                        accessTokenFactory: () => {
+                            return instancesConnectionInfo.accessToken ?? '';
```

Comment:
Do you know how long this token is valid for?

Reply:
I believe it is 1 hour. We will need to refresh it once it is expired.

Reply:
https://github.com/dotnet/AspNetCore.Docs/issues/13144 I found this interesting. They don't check if access token expires.

Reply:
What happens after 1 hour?  Do we stop receiving notifications?